### ISSUE TITLE
Error when trying to run checkstyle with Gradle

### ISSUE THREAD

**Issue by Sahilgat** [nus-cs2103-AY1920S1/forum#41]
Hi,

I'm getting the following error when using the "gradlew checkstyleMain checkstyleTest" command (Screenshot below):
Unable to instantiate 'MissingJavadocMethod' class, it is also not possible to instantiate it as .MissingJavadocMethod, MissingJavadocMethodCheck, .MissingJavadocMethodCheck. Please recheck that class name is specified as canonical name or read how to configure short name usage https://checkstyle.org/config.html#Packages. Please also recheck that provided ClassLoader to Checker is configured correctly.

![Capture](https://user-images.githubusercontent.com/38809578/64212746-f2043600-cedc-11e9-8d0e-9648a849b5a9.PNG)

When run from the command line, I get the following message:

Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details
> Task :checkstyleMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkstyleMain'.
> Unable to create Root Module: config {C:\Users\nisga\OneDrive\Desktop\duke\config\checkstyle\checkstyle.xml}, classpath {C:\Users\nisga\OneDrive\Desktop\duke\build\classes\java\main;C:\Users\nisga\OneDrive\De
sktop\duke\build\resources\main}.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
2 actionable tasks: 1 executed, 1 up-to-date

Could anyone advise on what may be causing the issue?

**Comment by geshuming** [comment]
https://github.com/nus-cs2103-AY1920S1/duke/blob/master/tutorials/gradleTutorial.md#checkstyle

Follow the tutorial exactly and include this config into your build.gradle

```
checkstyle {
    toolVersion = '8.23'
}
```

If you don't have the above config, the default tool version won't work with the checkstyle xml provided

**Comment by Sahilgat** [comment]
Yep silly mistake forgot to add the checkstyle config, thanks for your help!

**Comment by sheetalj2205** [comment]
@Sahilgat @geshuming Same issue I am facing even after changing the tool version, I don't have suppression.xml file though.
